src/ray/raylet/raylet.cc

```diff
@@ -24,10 +24,13 @@ Raylet::Raylet(boost::asio::io_service &main_service, const std::string &socket_
       acceptor_(main_service, boost::asio::local::stream_protocol::endpoint(socket_name)),
       socket_(main_service),
       object_manager_acceptor_(
-          main_service, boost::asio::ip::tcp::endpoint(boost::asio::ip::tcp::v4(), 0)),
+          main_service,
+          boost::asio::ip::tcp::endpoint(boost::asio::ip::tcp::v4(),
+                                         object_manager_config.object_manager_port)),
```

Comment:
This looks good: In `Raylet::RegisterGcs`, the node manager and object manager ports we submit to the GCS are read from the corresponding asio acceptor instances.